gammapy/analysis/core.py

```diff
@@ -270,8 +270,8 @@ def write_datasets(self, overwrite=True, write_covariance=True):
 
         Parameters
         ----------
-        overwrite : bool
-            overwrite datasets FITS files
```

Comment:
Here it was specified FITS file because the YAML files are always overwritten.

Reply:
I guess "overwrite" should apply to both, the YAML and FITS file?

Reply:
yes pushed a commit for that